sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/FileSASTests.groovy

```diff
@@ -24,11 +24,13 @@ class FileSASTests extends APISpec {
     private ShareFileClient primaryFileClient
     private ShareClient primaryShareClient
     private ShareServiceClient primaryFileServiceClient
+    private String shareName
 
-    private String shareName = "sharename"
     private String filePath = "filename"
 
     def setup() {
+        shareName = testResourceName.randomName(methodName, 60)
```

Comment:
Need to update all playback records in this file as this adds a new recording value that will be expected.